### PR TITLE
Add Coordinator and update interval settings

### PR DESCRIPTION
**Pull request by fwestenberg**
This pull request contains some new features, and a bug fix.

**New features**
- Coordinator class
- Simplified config flow
- Settings menu (see images below)
- Data is now stored in `entry.runtime_data` [docs](https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry/)

**Bug fix**
- Enable/disable the download of transaction data. My Alfen Single Pro Line stopped updating after some period because of the high load caused by the transaction download. 

)

)

### CODE REVIEW

**Comment by fwestenberg** [comment]
Hi @leeyuentuen thanks for all your great work! I hope you like this pull request. 
I changed the manifest version to "2.9.0"

**Comment by leeyuentuen** [comment]
Well done. I'll check and test them later the commit.

Good improvements

File: custom_components/alfen_wallbox/alfen.py [hunk 1]
```diff
@@ -78,71 +74,68 @@ def __init__(self,
...
-        disable_warnings()
+        # disable_warnings()
 
-        # Default ciphers needed as of python 3.10
-        context = ssl.create_default_context()
-        context.set_ciphers("DEFAULT")
-        context.check_hostname = False
-        context.verify_mode = ssl.CERT_NONE
-        self.ssl = context
+        # # Default ciphers needed as of python 3.10
```
Comment (fwestenberg) [inline custom_components/alfen_wallbox/alfen.py:87]: Why do we need to set SSL here? Seems like Home Assistant always sets the default

**Comment by fwestenberg** [comment]
@leeyuentuen Can you please wait before making a new hacs version? I have a very nice PR coming up.

**Comment by leeyuentuen** [comment]
normally i have create a new test release version

**Pull request merged**